### DISCUSSION TITLE
documentation/tutorial how to set up the infiltration information

### DISCUSSION THREAD

**Discussion by jennykupzig** [SynxFlow/SynxFlow#16] [Q&A] [answered]
It would be very nice to understand the infiltration module better to be able to set it up properly. What das the following input mean, and how to set it up, e.g., based on CORINE information.

```
sewer_sink : 0
cumulative_depth : 0
hydraulic_conductivity : 0
capillary_head : 0
water_content_diff : 0
```
As I could not find any tutorial covering this topic it would be very nice if an existing tutorial could be extended to this topic. 

Also it would be nice to know if the soil moisture could be also used as initial condition to account for different wetting conditions.

**Comment by xiaxilin** [comment]
Hi, setting these parameters would be the same as setting up values for the Manning's coefficient. Among those, the cumulative depth is the initial infiltrated depth that could be used as the initial moisture condition. I found a useful article that may be used for setting up the parameters

[Green-Ampt method](https://www.hec.usace.army.mil/confluence/rasdocs/ras1dtechref/6.1/overview-of-optional-capabilities/modeling-precipitation-and-infiltration/green-ampt#:~:text=The%20Green%2DAmpt%20(GA),a%20single%20piston%20like%20displacement.)

**Reply by jennykupzig** [reply]
Thanks for the link, actually I've found also a very informative one [https://wiki.tuflow.com/Green-Ampt_Infiltration_Parameters#Max_Ponding_Depth](url). There are explanation for all parameters and a small sensitivity analysis per parameter as well as tables for appropiate values for the parameters.

For TUFlow there are slightly different user parameters which can be however linked to synxflow (if I understand it correctly): 
- cumulative_depth (Initial Moisture in TUFlow)
- hydraulic_conductivity  (Saturated Hydraulic Conductivity in TUFlow)
- capillary_head  (Capillary Suction Head in TUFlow)
- water_content_diff (Porosity in TUFlow)

However, what I do not understand is, HOW I can define the cumulative depth as initial condition (for a simulation that is stopped and then started again) if the model does not provide it as model output. In my understanding this information should be also defined as model output (analogous to hUx, hUy and h). Or am I missing something?

**Reply by xiaxilin** [reply]
Thank you. The TUFLOW tutorial is very interesting. To set up the initial condition of cumulative_depth, you can use the same method for setting up Manning's coefficient using the case_input.set_grid_parameter() function.

**Reply by jennykupzig** [reply]
Thanks for the information that I can use `case_input.set_grid_parameter() function`! However, one question remains: 

How can I stop and re-start the model using the infiltration module when there is no cumulative depth at the end of a simulation written as model output which I can feed to the re-started model again? Am I missing something?

**Reply by xiaxilin** [reply, marked as answer]
Hi, currently it doesn't do this and a small change of source code is needed for this.

**Reply by jennykupzig** [reply]
Thanks for the clarification - I will create an issue for this, if its okay for you. :-)
